packages/@okta/vuepress-site/docs/guides/oie-upgrade-api-sdk-to-oie-sdk/main/java/signout.md

```diff
@@ -11,7 +11,7 @@ public String logout(final HttpSession session) {
 }
 ```
 
-### Okta Identity Engine SDK authentication flow for basic sign-out
+### Identity Engine SDK authentication flow for basic sign-out
```

Comment:
both headings:
Okta Authentication SDK authentication flow for basic sign-out
and
Identity Engine SDK authentication flow for basic sign-out

the style is to not use the term sign-in and sign-out as a noun. It's used as an adjective.

I struggled to figure out a better title that didn't repeat both authentication and flow

Reply:
Same fix as before. I updated them to

Okta Authentication SDK basic sign-out flow

and

Identity Engine SDK basic sign-out flow